association_backend/.env.sample

```diff
@@ -0,0 +1,13 @@
+FRONTEND_URL=http://localhost:8060/
```

Comment:
remove this, we are going to implement a better handling of .env files later

Reply:
ok